### ISSUE TITLE
Error of ore

### ISSUE THREAD

**Issue by GabrielitoPL** [Murdy007/Wizzy#13]
Hello, my friend, let me say that this pack is almost complete, and I like it, but there is a bug that is what I do not like about this pack, so that it's perfect, error is what it is, As for example, the copper, tin ores, etc .. that are repeated, and that is the fault, because I occupy a lot of space in my inventory, and would wish that they correct that error, that only a single type of ore to Do not take up so much inventory.

**Comment by Besleh** [comment]
sry late respons im looking over the forum not this thread :)
were aware of teh many diffrent ore types but if you setup a mekanism trippler the ingots will be from mekanism , however i will look into this as i noted/havving this problem too 

(altho i use a ender poch to my ME system) ^^ 
hope this helps /regards Zora

**Comment by Besleh** [comment]
might come with a hotfix patch as soon as i ggot all  the issues i  found ready

**Comment by GabrielitoPL** [comment]
Thank you very much, I am looking forward to the next version with the problem solved, because this pack is the most complete I have seen and I congratulate them, I love it very much: 3

**Comment by Besleh** [comment]
if you have more issues or sugguestions feel free to post em on the forum =)   / Zora

**Comment by Besleh** [comment]
i forwaded a solution so murdy its in his hands now =)

/Zora

**Comment by Besleh** [comment]
will take some time with config fixes but working on it / Zora

**Comment by GabrielitoPL** [comment]
Yes, I'm testing the mods just in case some fault, if I see something suspicious I'll tell you

**Comment by GabrielitoPL** [comment]
Seriously so much is the error !, It's okay to keep waiting: c

**Comment by Besleh** [comment]
its on its way / Zora

**Comment by GabrielitoPL** [comment]
huuuuuuuuuuuuuuuuuuuuuuu I can not stand it anymore, come on, come on now

**Comment by GabrielitoPL** [comment]
I want to play it now :,v

**Comment by Besleh** [comment]
the test version is out we are working on it will be released

**Comment by Murdy007** [comment]
Hi,
We have just released version 1.1.114, which includes an ore dictionary converter, where you can change ores into other counterparts from other mods. There is also the option to install TooManyDanyOres, which does it manually.
Thanks for playing!

**Comment by Besleh** [comment]
im using both ^^

**Comment by GabrielitoPL** [comment]
ajja Thanks a lot, now if you can say that I've been waiting for this update with ancias, sorry if I was a bit impatient to escrivirles, but it's been testing and if it works, and about the minerals I've gone crazy with the new mod function, which I did not know that I was going for a mod so, thank you very much, thank you and also the customer service that showed me that you are very good attendants, thank you very much. <3

**Comment by Besleh** [comment]
youre welcome (if you mean me =P) otherwise youre welcome

**Comment by GabrielitoPL** [comment]
another question, do you have any public servers to play with the mods?

**Comment by Besleh** [comment]
I was working on it but cant find anywhere to host it at the moment (im bad at those things :)
